ada/language/ast.py

```diff
@@ -6877,21 +6877,18 @@ def is_write_reference():
           2. `X (2) := 2;`
           3. `P(F => X)` where F is declared `out`.
           4. `X'Access`.
+          5. `X.C := 2`, `R.X := 2`
 
         .. note:: This is an experimental feature. There might be some
             discrepancy with the GNAT concept of "write reference".
         """
-        # TODO: Missing case for component assignment, etc. Also, this
-        # algorithm should probably be recursive. Think about::
-        #
-        #   A.x (2).y := 3
         return Entity.parent.match(
             # handle case 1
```

Comment:
Rather than referencing numbers in the docstring, having examples written in the comments literally would be a big help IMHO.